### DISCUSSION TITLE
Made glyphs can’t be seen

### DISCUSSION THREAD

**Discussion by NikitaGamer64** [fontforge/fontforge#4908] [Q&A]
I made a glyph like this:
![image](https://user-images.githubusercontent.com/36325411/152415502-bbe517d0-870e-49db-8459-2b2710cdc358.png)
But in chart, it isn’t visible:
![image](https://user-images.githubusercontent.com/36325411/152415608-93d04516-2630-44cc-be19-bf0ffd4f2590.png)
What should I do to see the glyph in the grid?

**Comment by ctrlcctrlv** [comment]
You imported an image, you didn't draw anything.

**Comment by NikitaGamer64** [comment]
How to convert the image then?

**Comment by ctrlcctrlv** [comment]
By drawing over it or using PoTrace.

Fonts consist of vector shapes (points in space and Bezier handles). They do not, in general, consist of bitmaps. If you want to make a bitmap font, you'd be doing that wrong as well, you'd want to use FontForge's BitmapView, not its CharView. I think  you go to “Add Bitmap Strike” but do check.
